### DISCUSSION TITLE
embark action for zk-new-note

### DISCUSSION THREAD

**Discussion by jgarte** [localauthor/zk#72] [Ideas]
WDYT of having an embark action for `zk-new-note`?

You make the selection and then you embark on it.

**Comment by localauthor** [comment]
Since embark works on specific targets, you would have to add `zk-new-note` to the `embark-region-map`:

`(define-key embark-region-map (kbd "N") 'zk-new-note)`

**Reply by jgarte** [reply]
Nice! That works really well ergonomically.

Can we add this to `zk`? I can open a PR if needed.

**Reply by localauthor** [reply]
I don't want `zk` to add commands to the default embark maps, because it could interfere with users' existing customizations. I would prefer to add a note in the readme, suggesting this as a possibility.
